### PR TITLE
feat(Popover): add prop to disable link styles

### PR DESCRIPTION
**Pull request by ogonkov**

### CODE REVIEW

File: src/components/Popover/Popover.tsx [hunk 1]
```diff
@@ -260,12 +263,22 @@ export class Popover extends React.Component<
...
             <Popup
                 anchorRef={anchorRef || this.controlRef}
-                className={b('tooltip', {theme, size, ['with-close']: hasClose}, tooltipClassName)}
+                className={b(
+                    'tooltip',
+                    {
+                        theme,
+                        size,
+                        ['with-close']: hasClose,
+                        force_links_appearance: forceLinksAppearance,
```
Comment (amje) [inline src/components/Popover/Popover.tsx:278]: `force-links-appearance`

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Popover/Popover.tsx (hunk 1 above)
Comment (ogonkov) [reply src/components/Popover/Popover.tsx]: ouch, fixed

**Review by ma-efremoff — APPROVED** [review]

File: src/components/Popover/Popover.tsx [hunk 2]
```diff
@@ -136,6 +138,7 @@ export class Popover extends React.Component<
         hasClose: false,
         initialOpen: false,
         disabled: false,
+        forceLinksAppearance: true,
```
Comment (ma-efremoff) [inline src/components/Popover/Popover.tsx:141]: It is better to have names of properties when you don't need to set any values in defaultProps. What do you think about `disableLinksAppearance`?

Comment (ogonkov) [reply src/components/Popover/Popover.tsx:141]: it looks like "inverted" prop for me, like having `notOpen` instead of `open`

Comment (ma-efremoff) [reply src/components/Popover/Popover.tsx:141]: Inverted for "notOpen" is 'closed', your example with negation, and I agree that having negation in names is a bad pattern. But there is no negation in names like `disableAnyThing`, also there is standard property `disabled`.

Comment (ogonkov) [reply src/components/Popover/Popover.tsx:141]: `disableLinksAppearance` will cause same negotiation logic


```js
if (!disableLinksAppearance) {
  //add modifier
}
```

Versus current


```js
if (forceLinksAppearance) {
  //add modifier
```

Comment (ma-efremoff) [reply src/components/Popover/Popover.tsx:141]: Negation is bad as part of a name, but it's ok for conditions. More important to have intuitive set props.

**Review by amje — APPROVED** [review]

**Pull request merged**